clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -381,41 +377,41 @@ public String toString() {
     }
 
     /**
-     * <p>This is used to stage the unsent {@link OffsetCommitRequestState} and {@link OffsetFetchRequestState}.
+     * <p>This is used to stage the unsent {@link OffsetCommitRequest} and {@link OffsetFetchRequestState}.
      * <li>unsentOffsetCommits holds the offset commit requests that have not been sent out</>
      * <li>unsentOffsetFetches holds the offset fetch requests that have not been sent out</li>
-     * <li>inflightOffsetFetches holds the offset fetch requests that have been sent out but incompleted</>.
-     *
+     * <li>inflightOffsetFetches holds the offset fetch requests that have been sent out but not completed</>.
+     * <p>
      * {@code addOffsetFetchRequest} dedupes the requests to avoid sending the same requests.
      */
 
     class PendingRequests {
         // Queue is used to ensure the sequence of commit
-        Queue<OffsetCommitRequestState> unsentOffsetCommits = new LinkedList<>();
+        Queue<OffsetCommitRequest> unsentOffsetCommits = new LinkedList<>();
         List<OffsetFetchRequestState> unsentOffsetFetches = new ArrayList<>();
         List<OffsetFetchRequestState> inflightOffsetFetches = new ArrayList<>();
 
-        public boolean hasUnsentRequests() {
+        boolean hasUnsentRequests() {
```

Comment:
Could this be private? Ditto below.

Reply:
Sorry - I left out commit.  Fixing those obvious mistakes right away.

Reply:
Sorry - leaving this protected as it is used in one of the test to verify there's no more unsent request.